cluster/src/main/scala/ai/forestflow/serving/cluster/ServableRegistry.scala

```diff
@@ -109,7 +110,10 @@ class ServableRegistry(localBasePath: String) extends Actor
   with HasPersistence {
   import ServableRegistry._
 
-//  implicit private val dispatcher: ExecutionContextExecutor = context.dispatcher
+  implicit private val system : ActorSystem = context.system
+  implicit private val dispatcher: ExecutionContextExecutor = context.dispatcher
+  implicit private val materializer: ActorMaterializer = ActorMaterializer()
+
   log.info(s"Using  dispatcher: ${context.dispatcher}")
```

Comment:
Given the dispatcher is now explicitly assigned to a variable, please use the variable "dispatcher" instead of context.dispatcher